components/support/rustlog/src/main/java/mozilla/components/support/rustlog/RustLog.kt

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.rustlog
+
+import mozilla.appservices.rustlog.LogLevelFilter
+import mozilla.appservices.rustlog.RustLogAdapter
+import mozilla.components.support.base.log.Log
+
+object RustLog {
+
+    /**
+     * Enable the rust log adapter. This does (almost) nothing if you are not
```

Comment:
This note (almost nothing) about megazord wasn't clear to me right away (before Nick explained it to me). Can we clarify this or simply write that it's only supported in Megazord for now?